conference.js

```diff
@@ -1473,6 +1473,7 @@ export default {
 
         return promise.then(
             () => {
+                APP.API.notifyScreenSharingStatusChanged(false);
```

Comment:
I wonder if it's necessary to have notifyScreenSharingStatusChanged fire for both success and fail cases of promise or somehow move the place of firing notifyScreenSharingStatusChanged. If didHaveVideo is true and switching to local video fails, right now notifyScreenSharingStatusChanged might not fire.

Reply:
@virtuacoplenny Indeed. After a more thorough code review, I think I found a better way to implement this notification. I've updated the branch